biome.json

```diff
@@ -4,9 +4,6 @@
     "enabled": true,
     "rules": {
       "recommended": true,
-      "performance": {
-        "noDelete": "off"
-      },
```

Comment:
This was a duplicate property. See `performance` below. I kept the one that has the most options set.